src/lib/storage/index/abstract_index.hpp

```diff
@@ -111,14 +113,30 @@ class BaseIndex : private Noncopyable {
   Iterator cbegin() const;
 
   /**
-   * Returns an Iterator past the position of the greatest indexed element. This is useful for open
+   * Returns an Iterator past the position of the largest indexed non-null element. This is useful for open
    * end range queries.
    * Iterating from cbegin() to cend() will result in a position list with ordered values.
    * Calls _cend() of the most derived class.
    * @return an Iterator on the end of the index (one after the last element).
    */
   Iterator cend() const;
 
+  /**
+   * Returns an Iterator to the position of the first element equal to null.
```

Comment:
Add a comment that NULL is supported for single-column indexes only and that we do not yet have an idea on how to handle multi-columns?